lib/ansible/modules/utilities/logic/wait_for.py

```diff
@@ -93,6 +93,10 @@
     choices: [ "present", "started", "stopped", "absent", "drained" ]
     required: False
     default: "started"
+  ignore_time_wait:
+    description:
+      - ignore time_wait socket states when waiting for connections to be drained
+    required: false
```

Comment:
Makes sense...  Something like an "active_connection_states" option which takes a list and has the following default:
[ 'ESTABLISHED', 'SYN_SENT', 'SYN_RECV', 'FIN_WAIT1', 'FIN_WAIT2', 'TIME_WAIT' ]
?